docker-compose.yml

```diff
@@ -1,40 +1,29 @@
 version: '3.7'
-services:
-  offline:
-    image: aligent/serverless:offline
-    networks:
-      - localstack-net
-    command: tail -f /dev/null
-    volumes:
-      - .:/app
-      - ~/.npm:/home/node/.npm
-      - ~/.aws:/home/node/.aws
 
+services:
   localstack:
-    image: localstack/localstack:0.12.7
+    image: localstack/localstack:1.0.4
+    container_name: localstack-debug
     networks:
-      - localstack-net
-    ports:
-      - '4566:4566'
-      - '4571:4571'
+      localstack-net:
+        ipv4_address: ${AWS_LOCALSTACK:-172.20.0.100}
```

Comment:
How is this going to behave with multiple environments? If we have multiple with the same IP that could cause problems. Something we've been looking at doing in DevOps for a while now is splitting local stack into it's own repo and sharing localstack environments between services. Now might be a good time to look into that a bit further...

Reply:
@TheOrangePuff I'm not really understand the question and the problem we have here. Maybe we can have a look together to find the best solution? My approach here with this is having localstack running on it own (like a virtual AWS service). Once it's running, it create it own's network where other machine can join. At the time we want to deploy a lambda function for example, we spin up another container, join the existing localstack network & deploy to the localstack container.